libs/components/src/lib/interview-modal/interview-modal.spec.tsx

```diff
@@ -0,0 +1,28 @@
+import { unmountComponentAtNode } from 'react-dom';
+import { render } from '@testing-library/react';
+
+import InterviewModal from './interview-modal';
+
+describe('InterviewModal', () => {
+  const container: Element | DocumentFragment | null =
+    document.createElement('div');
+  beforeEach(() => {
+    document.body.appendChild(container);
+  });
+
+  afterEach(() => {
+    unmountComponentAtNode(container);
+    container.remove();
+  });
+  it('should render successfully', () => {
```

Comment:
`'should render successfully'` what and when?

Reply:
fixed

Reply:
I dont see changes

Reply:
changed